hack/e2e-test.sh

```diff
@@ -86,16 +86,25 @@ DEPLOY_IMAGE="${HIVE_IMAGE}" make deploy
 function teardown() {
 	echo ""
 	echo ""
+        # Skip tear down if the clusterdeployment is no longer there
+        if ! oc get clusterdeployment | grep ${CLUSTER_NAME}; then
```

Comment:
Can this just search for the specific clusterdeployment instead of doing a grep?
```suggestion
        if ! oc get clusterdeployment ${CLUSTER_NAME}; then
```